Client/pings.yaml

```diff
@@ -18,3 +18,13 @@ topsites-impression:
     - https://github.com/mozilla-mobile/firefox-ios/pull/10696
   notification_emails:
     - fx-ios-data-stewards@mozilla.com
+first-session:
+  description: |
+    Recorded on first_session when the user installs the app.
+  include_client_id: false
```

Comment:
Since the metrics above don't include any kind of hardware identifier or other seemingly personalized identifier, I don't think you need to exclude the client_id here.  The reason Fenix did this was because they were sending a hashed version of the Google Advertising identifier and we didn't want to link the GAiD with our client_id for privacy reasons.

If we can include the client_id here, it will make it much easier to link this attribution data to other telemetry data in analysis.